pepr/privacy/mia.py

```diff
@@ -0,0 +1,1182 @@
+                    tab_ap.add_row(["Attack batch size", ap["attack_batch_size"]])
+                    tab_ap.add_hline()
+                self.report_section.append(Command("captionsetup", "labelformat=empty"))
+                self.report_section.append(
+                    Command(
+                        "captionof",
+                        "table",
+                        extra_arguments="Attack parameters",
+                    )
+                )
+
+            with self.report_section.create(MiniPage(width=r"0.49\textwidth")):
+                # -- Create table for the data configuration
+                self.report_section.append(Command("centering"))
+                nr_targets, target_training_set_size = dc[
+                    "record_indices_per_target"
+                ].shape
+                with self.report_section.create(Tabular("|l|c|")) as tab_dc:
+                    tab_dc.add_hline()
+                    tab_dc.add_row(
+                        [
+                            NoEscape("Samples used to train shadow models ($S$)"),
+                            len(dc["shadow_indices"]),
+                        ]
+                    )
+                    tab_dc.add_hline()
+                    tab_dc.add_row(
+                        [
+                            NoEscape("Samples used to train target models ($T$)"),
+                            len(dc["target_indices"]),
+                        ]
+                    )
+                    tab_dc.add_hline()
+                    tab_dc.add_row(
+                        [
+                            "Samples used to evaluate the attack",
+                            len(dc["evaluation_indices"]),
+                        ]
+                    )
+                    tab_dc.add_hline()
+                    tab_dc.add_row(["Attacked target models", nr_targets])
+                    tab_dc.add_hline()
+                    tab_dc.add_row(
+                        ["Target model's training sets size", target_training_set_size]
+                    )
+                    tab_dc.add_hline()
+                    tab_dc.add_row(
+                        [
+                            NoEscape("Size of $S \cap T$"),
+                            len(set(dc["shadow_indices"]) & set(dc["target_indices"])),
+                        ]
+                    )
+                    tab_dc.add_hline()
+                self.report_section.append(Command("captionsetup", "labelformat=empty"))
+                self.report_section.append(
+                    Command(
+                        "captionof",
+                        "table",
+                        extra_arguments="Target and Data Configuration",
+                    )
+                )
+
+    def _report_attack_results(self, save_path):
+        """
+        Create subsubsection describing the most important results of the attack.
+
+        This subsection contains results only for the first target model.
+        """
```

Comment:
We should add the parameter list to the docstring :)

Reply:
Fixed in 03c76ef